packages/core/lib/testing/TestSource.js

```diff
@@ -97,6 +97,19 @@ module.exports = class TestSource {
         return { body, filePath: importPath };
       }
     }
+
+    const loggingLibraries = ["Console"];
+
+    for (const lib of loggingLibraries) {
+      const actualImportPath =
+        typeof BUNDLE_VERSION !== "undefined"
+          ? path.resolve(__dirname, path.basename(importPath))
+          : path.resolve(__dirname, "../logging", path.basename(importPath));
+      if (importPath === `truffle/${lib}.sol`) {
+        const body = fse.readFileSync(actualImportPath, { encoding: "utf8" });
+        return { body, filePath: importPath };
+      }
+    }
```

Comment:
Did I accidentally add this here when you removed it @eggplantzzz?

Reply:
No, I think this is where this was.

Reply:
Hm, then seems it's duplicated by your Truffle resolver source, no? Is there a way to remove this duplication?

Reply:
Ok let me check on it again to see which is correct. Maybe I'm mistaken

Reply:
It is duplicated but I'm not sure it is worth it to deal with such a small amount of code. Anything come to mind?

Reply:
It just makes me wonder... is your Truffle ResolverSource even getting used?

Reply:
Currently, it looks to me like `TestSource` will always resolve this logging library and it will never get it from `resolver`.

Reply:
So the conclusion here was to remove `TestSource` altogether since that is now handled by the `truffle` source.